framework/src/mysql.js

```diff
@@ -13,6 +13,8 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+// TODO: Rename all methods to use 'DB' instead of 'Db'
```

Comment:
TODO should be resolved?

Reply:
No, we'll be resolving it as part of the `0.7.0` release.